.mocharc.yml

```diff
@@ -2,8 +2,6 @@ colors: true
 file:
   - test/test_helper.ts
 full-trace: true
-forbid-only: true
```

Comment:
I think these config changes should not be reverted, just something that got picked up during that original PR.